src/gui/touchscreengui.cpp

```diff
@@ -849,11 +855,14 @@ void TouchScreenGUI::translateEvent(const SEvent &event)
 					m_camera_pitch = MYMIN(MYMAX(m_camera_pitch + (dy * d), -180), 180);
 
 					// update shootline
+					if (m_draw_crosshair) {
```

Comment:
It looks like m_shootline is not use at all if m_draw_crosshair is true. Is it necessary to make sure it has a correct value?

Reply:
The shootline is updated elsewhere, in the same place desktop does it

Reply:
I do not know. I wrote it just for consistency with other code that is there. I am guessing that it is not necessary for this very case.